plugins/webp-uploads/readme.txt

```diff
@@ -60,6 +60,16 @@ By default, the Modern Image Formats plugin will only generate WebP versions of
 
 == Changelog ==
 
+= 2.3.0 =
+
+**Enhancements**
+
+* Introduce `webp_uploads_get_file_mime_type` helper function. ([1642](https://github.com/WordPress/performance/pull/1642))
```

Comment:
I don't think we should include this in the changelog. The function should really be `@access private` anyway. I'm changing this to `skip-changelog`.

Reply:
See bd422e9

Reply:
I think this is reasonable to include since it's a new function. I disagree this should be `@access private`.

To be fair, none of the plugin's functions is really intended for external usage, but then we should rather re-consider our annotations and e.g. always use `@access private` if we think that makes sense - but doing it just here would be wrong to me.